### DISCUSSION TITLE
Save query in the search bar after quitting

### DISCUSSION THREAD

**Discussion by strozek-xx** [quodlibet/quodlibet#3527] [Ideas]
Using v4.3.0 on Linux, my most used query in the search bar is '#(added < time period)'. When QL is closed the search bar is emptied. It would be nice if QL saves the most recent search query so that in this case recently added tracks are displayed upon opening QL.

**Comment by bassstorm** [comment]
Hi @strozek-xx ,

Sorry for late reply.
Do you switch browsers at app startup (using [cli params](https://quodlibet.readthedocs.io/en/latest/guide/commands/quodlibet.html) for example)? If not, I cannot reproduce your issue on QL 4.4, for me search query is saved and loaded at startup.

Thanks!

**Reply by strozek-xx** [reply]
Hi bassstorm, 
Thank you for the reply, i only see it now.
Switching between Internet radio browser and Panel browser and the search query disappears. 
A Quit from the Panel browser saves the search query, i'll use that as a workaround then. 
Besides this minor inconvenience i'm a fan and long-time user of QL, thank you for the great work !

**Reply by declension** [reply]
Hi - perhaps raise a bug if the iradio is not saving queries properly
